### PR TITLE
Add events links and individual template. Change event-scraper behavi…

### PR DESCRIPTION
**Pull request by amed**
…our to save files into source/events

TODO:
* [ ] archive navigation:  month names need to be localized to `en`

### CODE REVIEW

**Comment by jhilden** [comment]
Deploy preview ready!

Built with commit db982e75e86c977fc1412afa4c24e66101d39d5c

https://deploy-preview-22--webmontag.netlify.com

**Comment by amed** [comment]
It will be better that we make the TODO:
   archive navigation: month names need to be localized to 'en'
in another ticket.
This branch has already covered many issues.
The goal of covering many issues was because of issues-overlap and one-track thinking (speaking about myself)

File: lib/events-scraper/event.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def store_file
     # check if the events directory is exists.
     Dir.mkdir(EVENTS_DIR) unless File.exists?(EVENTS_DIR)
     # store the events files.
-    File.open("#{EVENTS_DIR}#{@date_string}.html", "w:UTF-8") do |f|
+    File.open("#{EVENTS_DIR}#{@date_string}.html.erb", "w:UTF-8") do |f|
```
Comment (jhilden) [inline lib/events-scraper/event.rb:55]: why do those files have `.erb` as the file extension?

File: source/assets/stylesheets/events.css [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
 	border-bottom: 1px solid #333;
 }
-
-.archive{
+.past{
+  background-color: #ededed;
   clear: right;
   float: right;
   width: 33%;
+  padding: 0;
```
Comment (jhilden) [inline source/assets/stylesheets/events.css:14]: :+1: for propper indenting :)

File: source/assets/stylesheets/style.css [hunk 3]
```diff
@@ -16,7 +16,8 @@ body {
 	height: 100%;
 	font-family: Lato, serif;
 	font-weight: 400;
-	font-size: 17px; /* 16px to 10px */
+  color: #191919;
+	font-size: 17px;
```
Comment (jhilden) [inline source/assets/stylesheets/style.css:20]: here we have the wrong indentation again

File: source/shared/content/_archive.slim [hunk 4]
```diff
@@ -1,6 +1,14 @@
 h2 Archive
-ul
-  -past_events.each_with_index do |past_event|
-    li
-      a href="#"
-        ="#{past_event.date}"
+
+- #event.date[0,4] is event.date.year
+-grouped_events = past_events.group_by{ |event| event.date[0,4] }.values
+-grouped_events.each do |a_year_events|
```
Comment (jhilden) [inline source/shared/content/_archive.slim]: wording suggestions:
`grouped_events` => `events_grouped_by_year`
`a_year_events` => `year_events_group`

Comment (jhilden) [reply source/shared/content/_archive.slim]: important:  this is _only a suggestion_.  ultimately you decide what makes most sense to you.

File: source/shared/content/_archive.slim [hunk 5]
```diff
@@ -1,6 +1,14 @@
...
+  .year-wrap
+    h4
+      ="#{a_year_events[0].date[0,4]}" unless a_year_events.empty?
+    ul
+      -a_year_events.each do |a_year_event|
+        li
+          - #date[5..-1] is "month-day"
```
Comment (jhilden) [inline source/shared/content/_archive.slim]: what is this comment for?

File: source/shared/index/_intro.slim [hunk 6]
```diff
@@ -2,7 +2,7 @@
 	a name="intro-en"
 
 	p
-		a href="#intro-de" onclick="javascript:$('#content-en').hide();$('#content-de').show();eventBox.resizeTriangle();" Zur Deutschen Version
+		a class="change-language" href="#intro-de" onclick="javascript:$('#content-en').hide();$('#content-de').show();eventBox.resizeTriangle();" Zur Deutschen Version
```
Comment (jhilden) [inline source/shared/index/_intro.slim]: CSS classes should describe something about the **visual** properties of an element not the functional properties.  can you think of a better name?

**Review by jhilden — APPROVED** [review]

File: lib/events-scraper/event.rb (hunk 1 above)
Comment (amed) [reply lib/events-scraper/event.rb:55]: When it has no .erb extension, it's get treated like normal file on the server.
That means it renders no layouts.

I will add the reason of .erb as a comment in event-scraper.

Comment (amed) [reply lib/events-scraper/event.rb:55]: I will fix this in another branch, this branch is called "indentation-and-erb"

File: source/assets/stylesheets/style.css (hunk 3 above)
Comment (amed) [reply source/assets/stylesheets/style.css:20]: I will fix this in another branch, this branch is called "indentation-and-erb"

File: source/shared/content/_archive.slim (hunk 4 above)
Comment (amed) [reply source/shared/content/_archive.slim]: I think what you suggest now is better. It is more readable to everyone.

File: source/shared/content/_archive.slim (hunk 5 above)
Comment (amed) [reply source/shared/content/_archive.slim]: Forgotten child

Comment (amed) [reply source/shared/content/_archive.slim]: It shows that this sub of the string equals to "month-day"

File: source/shared/index/_intro.slim (hunk 6 above)
Comment (amed) [reply source/shared/index/_intro.slim]: Done! by removing the whole class.

File: source/shared/content/_archive.slim (hunk 5 above)
Comment (amed) [reply source/shared/content/_archive.slim]: There is no need for it anyway.

**Pull request merged**